parser/parser_test.go

```diff
@@ -331,7 +331,19 @@ var parseTests = []parseTest{
 	{
 		"SimpleService",
 		"service TestService {\n  rpc Foo(In) returns (Out);\n}\n message In{} message Out{}",
-		`service { name: "TestService" method { name:"Foo" input_type:".In" output_type:".Out" } }` +
+		`service { name: "TestService" method { name:"Foo" input_type:".In" output_type:".Out" client_streaming: false server_streaming: false } }` +
+			`message_type:{name:"In"} message_type:{name:"Out"}`,
+	},
+	{
+		"SimpleServiceWithMessageCalledStream",
```

Comment:
My intention here is to test the special case where there's a message called "stream". I did this because my first version didn't handle this special case and it didn't seem obvious to me. This could just be me though. I took a look at parser_unittest.cc and it doesn't look like it tests for this special case.